src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -110,3 +111,72 @@ def test_method_read_status(session):
     assert isinstance(backlog, int)
     assert backlog == 0
 
+
+def test_fetch_error_while_not_initiated(session):
+    try:
+        session.fetch(1000)
+        assert False
+    except nidmm.Error as e:
+        assert e.code == -1074118641   # called fetch before calling Initiate or after calling Abort
+
+
+def test_multi_point_acquisition_with_measurement_absolute(session):
+    session.configure_multi_point(4, 2)
+    session.configure_measurement_absolute(nidmm.Function.DC_VOLTS, 0.02, 0.001)
+    measurements, numberOfMeasurements = session.read_multi_point(8)
+    assert len(measurements) == 8
+    assert numberOfMeasurements == 8
+
+
+def test_disable(session):
+    session.configure_measurement_digits(nidmm.Function.DC_VOLTS, 10, 5.5)
+    with session.initiate():
+        time.sleep(0.1)
+        backlog, acquisition_state = session.read_status()
+        assert acquisition_state == nidmm.AcquisitionStatus.FINISHED_WITH_BACKLOG
+        session.disable()
+        time.sleep(0.1)
+        backlog, acquisition_state = session.read_status()
+        assert acquisition_state == nidmm.AcquisitionStatus.NO_ACQUISITION_IN_PROGRESS
+
+
+def test_fetch_multiple(session):
+    session.configure_measurement_digits(nidmm.Function.DC_VOLTS, 10, 5.5)
+    session.configure_multi_point(sample_count=10, trigger_count=1)
+    iteration = 0
+    with session.initiate():
+        while True:
+            time.sleep(0.1)
+            backlog, acquisition_state = session.read_status()
+            measurements = session.fetch_multi_point(5)
```

Comment:
Why are we waiting? Why not fetch with the timeout to do the waiting?
What does read status do in this case, we never use it?